Dockerfile

```diff
@@ -47,23 +32,20 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 COPY scripts scripts
 RUN bash scripts/install_onnx_gpu_for_amd.sh && \
     bash scripts/install_torch_amd.sh && \
-    bash scripts/install_redis_centos.sh && \
-    mkdir -p /etc/redis && \
-    touch /etc/redis/redis.conf && \
-    echo "echo never > /sys/kernel/mm/transparent_hugepage/enabled" >> /etc/rc.local && \
-    echo "save ''" >> /etc/redis/redis.conf
+    bash scripts/install_redis.sh && \
+    bash scripts/install_ffmpeg.sh && \
+    bash scripts/install_punkt_tokenizers.sh
 
-ADD scripts/start_vespa.sh /usr/local/bin/start_vespa.sh
+# Install Vespa and pin the version. All versions can be found using `dns list vespa`
+# This is installed as a separate docker layer since we need to upgrade vespa regularly
+RUN dnf config-manager --add-repo https://raw.githubusercontent.com/vespa-engine/vespa/master/dist/vespa-engine.repo && \
+    dnf install -y vespa-8.396.18-1.el8
 
-# Install Vespa
-RUN dnf config-manager \
-  --add-repo https://raw.githubusercontent.com/vespa-engine/vespa/master/dist/vespa-engine.repo &&\
-    dnf config-manager --enable powertools &&\
-    dnf install -y epel-release &&\
-    dnf install -y vespa
+ADD scripts/start_vespa.sh /usr/local/bin/start_vespa.sh
 
 # Set Envs for Vespa
 ENV PATH="/opt/vespa/bin:/opt/vespa-deps/bin:${PATH}"
+# TODO check if following env vars are required
```

Comment:
@wanliAlex , are these two env vars used? `VESPA_LOG_STDOUT` and `VESPA_LOG_FORMAT`?

Reply:
I'll leave them here for now. and remove them in the next release of marqo-base image